enterprise/utils.py

```diff
@@ -399,6 +398,64 @@ def get_enterprise_customer_for_user(auth_user):
     return enterprise_customer_user.enterprise_customer
 
 
+def get_enterprise_customer_user(user_id, enterprise_uuid):
+    """
+    Return the object for EnterpriseCustomerUser.
+
+    Arguments:
+        user_id (str): user identifier
+        enterprise_uuid (UUID): Universally unique identifier for the enterprise customer.
+
+    Returns:
+        (EnterpriseCustomerUser): enterprise customer user record
+    """
+    try:
+        # pylint: disable=no-member
+        enterprise_customer_user = enterprise.models.EnterpriseCustomerUser.objects.get(
+            enterprise_customer__uuid=enterprise_uuid,
+            user_id=user_id
+        )
+    except enterprise.models.EnterpriseCustomerUser.DoesNotExist:
+        return None
+
+    return enterprise_customer_user
+
+
+def user_requires_data_sharing_consent(enterprise_customer_user, course_id=None):
```

Comment:
nit: name of the functions seems a bit mis-leading. It seems as if user requires something as opposed to user is required to do something. I think we should rename it to something like `require_user_content_for_sharing` or `require_user_content_for_data_sharing`or something similar.